android/app/src/main/AndroidManifest.xml

```diff
@@ -41,26 +41,12 @@
         </activity>
         <activity android:name="com.facebook.react.devsupport.DevSettingsActivity"/>
 
-        <service android:name="io.invertase.firebase.messaging.RNFirebaseMessagingService">
-            <intent-filter>
-                <action android:name="com.google.firebase.MESSAGING_EVENT" />
-            </intent-filter>
-        </service>
         <service android:name="io.invertase.firebase.messaging.RNFirebaseInstanceIdService">
             <intent-filter>
                 <action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
             </intent-filter>
         </service>
 
-        <meta-data
```

Comment:
Do we not need this anymore?

Reply:
It is perfectly possible that this is now automatically included in the manifest  due to the new dependencies